src/types.ts

```diff
@@ -274,6 +277,50 @@ export type GeckoCodeType = {
   contents: Uint8Array;
 };
 
+export enum FodPlatformSide {
+  RIGHT = 0,
+  LEFT = 1,
+}
+
+export type FodPlatformType = {
+  frame: number;
```

Comment:
```suggestion
  frame: number | null;
```

all the new type fields need a similar change